pqlite/index.py

```diff
@@ -74,6 +76,7 @@ def __init__(
         self._use_smart_probing = True
 
         self.read_only = read_only
+        self.logger = logger_flag
```

Comment:
I would reject this change about logging. For the logging, we should use logging API `logger.setLevel()` to turn off debug/info message.

Reply:
Now, the default level is `DEBUG`, we can change default as `INFO`

Reply:
Removed the logger flag